### PR TITLE
Feature j8583 message

### PR DESCRIPTION
**Pull request by gaugerard**

### CODE REVIEW

File: src/test/java/io/github/project/J8583MessageTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import jakarta.xml.bind.DatatypeConverter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+class J8583MessageTest {
+
+    @Test
+    void test() {
```
Comment (lorenzoDevOn) [inline src/test/java/io/github/project/J8583MessageTest.java:12]: review test name

Comment (gaugerard) [reply src/test/java/io/github/project/J8583MessageTest.java:12]: DONE

**Review by lorenzoDevOn — COMMENTED** [review]
fix comments

**Review by lorenzoDevOn — CHANGES_REQUESTED** [review]
fix comments

**Review by lorenzoDevOn — APPROVED** [review]

**Pull request merged**